CHANGELOG.md

```diff
@@ -8,6 +8,29 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Tractus-X has switched to YY.0M.MICRO [Calendar Versioning](https://calver.org/) for overarching, quarterly Releases. Semantic Versioning remains unchanged for all components.
 
+## [24.03] - 2024-03-XX
+
+### Added
+### Changed
+### Unchanged, untested
+### Known knowns
```

Comment:
Relevant components and interfaces developed considering Gaia-X Trust Framework - 22.10
Connection to an external Clearing House service for the entire Catena-X ecosystem is required. The Gaia-X compliance service should generally be capable of recognizing all companies that are able to register with the Catena-X Portal.
_Conformity to [Asset Administration Shell v3.0](https://industrialdigitaltwin.org/wp-content/uploads/2023/04/IDTA-01002-3-0_SpecificationAssetAdministrationShell_Part2_API.pdf), [API live example](https://app.swaggerhub.com/apis/Plattform_i40/AssetAdministrationShellRegistryServiceSpecification/V3.0_SSP-001#/Asset%20Administration%20Shell%20Registry%20API/GetAssetAdministrationShellDescriptorById) (AAS) is given. For compatibility reasons, an older version of the DTR query interface “/query” will additionally be supported until the end of 2023. Details can be found in the [changelog](https://github.com/eclipse-tractusx/sldt-digital-twin-registry/blob/main/CHANGELOG.md) of the respective component._

Remaining Useful Life (RUL) KIT was not verified in combination with the current Release package and is only compatible with Catena-X Standards as specified. Use at your own risk at this time.
FOSS components developed and pre-tested to TRL6 with artificial test data only
No crosscheck of functionality in verification environment (STABLE, Pre-PROD)
No execution of Load-, Performance- or Penetration Tests
No assignment of Export Control Classification Numbers (ECCN) to FOSS components
GeoBlocking recommended for Operations (GBaaS)
Security concept (overall) recommended for Operations